salt/states/pkgrepo.py

```diff
@@ -333,7 +332,7 @@ def managed(name, ppa=None, **kwargs):
             if enabled is not None \
             else salt.utils.is_true(disabled)
 
-    elif os_family in ('redhat', 'suse'):
+    elif __grains__['os_family'] in ('RedHat', 'Suse', 'SUSE'):
```

Comment:
The all uppercase `os_family` "SUSE" is invalid, it is only 'Suse'. If someone is using something else, then this is wrong.

Reply:
I was looking at the grains source to figure out what all possible values could be, because I didn't know the exact capitalization since the original code used .lower().